spec/scenic/schema_dumper_spec.rb

```diff
@@ -36,4 +36,48 @@ class Search < ActiveRecord::Base; end
       Search.connection.drop_view :'scenic.searches'
     end
   end
+
+  context "with dependent views " do
+    it "dumps create_view sorted alphabetically" do
+      Search.connection.execute("CREATE SCHEMA scenic")
+      Search.connection.execute("SET search_path TO scenic, public")
```

Comment:
These aren't undone after this test, so I think you've got a leaky test here. Do we need the scenic schema?